README.md

```diff
@@ -33,6 +33,47 @@ entropy coding, by clicking the thumbnail (original tracks provided by [Lucille
 <img src="./thumbnail.png" alt="Thumbnail for the sample video.
 	You will first here the ground truth, then ~3kbps, then 12kbps, for two songs."></a></p>
 
+## 🤗 Transformers
+
+Encodec has been added to Transformers. For more information, please refer to [Transformers' Encodec docs](https://huggingface.co/docs/transformers/main/en/model_doc/encodec).
+
+You can find both the [24KHz](https://huggingface.co/facebook/encodec_24khz) and [48KHz](https://huggingface.co/facebook/encodec_48khz) checkpoints on the 🤗 Hub.
+
+### Usage
+
+You can use Encodec at scale with massive datasets with less than 10 lines of code! ⚡️
```

Comment:
Maybe rephrase as "Using Transformers smiley, you can use Encodec at scale along with all the other supported model and datasets. Alternatively you can also directly the `encodec` package, as detailed in the Usage section."

While I like Transformers I don't want to make it look like it is the only option! For some use cases the original encodec might be much lighter to install and use!

Reply:
Definitely! I just copied your phrasing 1:1 with minor changes, hope that works!